csgo_bot.py

```diff
@@ -96,6 +96,24 @@ def list_mix(message):
 def test(message):
     print("TEST")
 
+def get_javigon(message):
+    chat_id = message.chat.id
+    bot.send_audio(chat_id=chat_id, audio=open('javigon(by ZEN).ogg', 'rb'))
```

Comment:
Wouldn't it be better to have a file name w/o spaces or parenthesis?

PD: Are the audio files included in the commit?

Reply:
yes